lib/p2p/Pool.ts

```diff
@@ -214,7 +214,7 @@ class Pool extends EventEmitter {
     const peer = this.peers.get(nodePubKey);
     if (peer) {
       peer.close();
-      this.logger.info(`Disconnected from ${peer.nodePubKey} @ ${addressUtils.toString(peer.socketAddress)}`);
+      this.logger.info(`Disconnected from <${peer.nodePubKey}>@${addressUtils.toString(peer.socketAddress)}`);
```

Comment:
I don't think we actually want the `<>` surrounding the nodepubkey, but it would be good to remove the spaces on either side of the `@` as you've done.